Dockerfile

```diff
@@ -1,35 +1,35 @@
-FROM ubuntu-debootstrap:14.04
+FROM php:7.0.7-alpine
 MAINTAINER Christian Lück <christian@lueck.tv>
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
-  nginx supervisor php5-fpm php5-cli \
-  php5-pgsql php5-mysql php5-sqlite php5-mssql \
-  wget
+## Install php and dependencies
+RUN set -ex \
```

Comment:
well, at least the `-e` part does not seems very useful as each commands are joined with `&&`. I'm not sure about the `'x` part. it's used to print the execution trace, that's it ?
tbh, it's just for curiosity as I don't think it has any impact on the image in the end